src/components/SendFundsForm.vue

```diff
@@ -527,7 +527,14 @@ export default {
           () =>
             isErc20(this.currency)
               ? this.ethBalance >= this.transferFee || this.$t('transfer.error_not_enough_eth_fee')
-              : true
+              : true,
+          (v) => {
+            const isKlyTransfer = this.currency === Cryptos.KLY
+            if (!isKlyTransfer) return true
+            const MAX_UINT64 = BigInt('18446744073709551615')
```

Comment:
Can you import it from Klayr lib?
https://github.com/klayrHQ/klayr-sdk/blob/development/elements/validator/src/constants.ts#L19

Reply:
No, otherwise I would import it.

Reply:
Try this

```js
import { MAX_UINT64 } from '@klayr/validator'
```